apps/src/dance/blocks.js

```diff
@@ -113,9 +114,9 @@ export default {
         FUNCTION_NAME_LABEL: i18n.behaviorEditorLabel(),
         FUNCTION_DESCRIPTION_LABEL: i18n.behaviorEditorDescription(),
       },
-      'behavior_definition',
+      BLOCK_TYPES.behaviorDefinition,
```

Comment:
Fun fact: Behaviors as a concept were originally intended for use in both Dance and Sprite Lab, but we never released anything using them in the former. IMHO, we should consider deleting any code that references them here.